src/ciphersuite/codec.rs

```diff
@@ -84,3 +84,13 @@ impl Codec for Secret {
         Ok(Secret { value })
     }
 }
+
+impl KdfLabel {
+    pub fn serialize(&self) -> Vec<u8> {
+        let mut buffer = Vec::new();
+        (self.length as u16).encode(&mut buffer).unwrap();
```

Comment:
Ugh, I missed this. These should probably not be `unwrap`s but `?`.

Reply:
At the risk of sounding repetitive, I'd like to chalk this up to #228, as this is another point, where we have to bubble up errors from key derivation functions.

Reply:
Ok, sound like you know what's your next issue. is 😉